core/core/services/birthday.py

```diff
@@ -1,9 +1,65 @@
 from typing import Any
-from datetime import date
+from datetime import date, datetime, timedelta
 from core.models import response, Record
+from core.database import write_data, Database
+
+database = Database()
+
+
+@response()
+@write_data
+def add_birthday(payload):
+    record = database[payload.name]
+    birthday = get_valid_birthday(payload.birthday)
+
+    return set_birthday(payload, record, birthday)
+
+
+@response()
+def get_birthdays_by_duration(payload):
+    records = database.all()
+    today = date.today()
+    records_with_this_week_birthday: list[Record] = []
+    days = int(payload.day_amount)
```

Comment:
lets make `payload.day_amount` as `int`, so we don't need to convert it here

Reply:
Done